modules/LookUpGBenchmark.cmake

```diff
@@ -1,4 +1,11 @@
 # Installs GBenchmark into build directory
+if(GBenchmark_ARGUMENTS)
+  cmake_parse_arguments(GBenchmark "" "GIT_REPOSITORY" "" ${GBenchmark_ARGUMENTS})
```

Comment:
What the first double quotes are for ?

Reply:
cmake_parse_arguments expects 5 arguments, and the "options" one is empty.
See https://cmake.org/pipermail/cmake/2012-October/052443.html

Reply:
hm ok make looks pretty conservative in its syntax. Sounds good to me. Can be merged